sdcm/fill_db_data.py

```diff
@@ -783,7 +783,7 @@ class FillDatabaseData(ClusterTester):
                 "INSERT INTO nameless_index_test (id, birth_year) VALUES ('Paul', 24)",
                 "INSERT INTO nameless_index_test (id, birth_year) VALUES ('Bob', 42)"],
             'queries': ["SELECT id FROM nameless_index_test WHERE birth_year = 42"],
-            'results': [[['Bob'], ['Tom']]],
+            'results': [[['Tom'], ['Bob']]],
```

Comment:
Please add a comment above, mention that the order has changed in 4.4.

Reply:
Done